backend/src/org/commcare/util/CommCarePlatform.java

```diff
@@ -437,4 +438,22 @@ public String getModuleNameForEntry(Entry entry) {
         }
         return resolved;
     }
+
+    public String getMenuDisplayStyle(String menuId) {
+        Vector<Suite> installed = getInstalledSuites();
+        String commonDisplayStyle = null;
+        for(Suite s : installed) {
+            for(Menu m : s.getMenus()) {
+                if(menuId.equals(m.getId())) {
+                    if(m.getStyle() != null) {
+                        if(commonDisplayStyle != null && !m.getStyle().equals(commonDisplayStyle)){
```

Comment:
This is what is covered in the mixed-menu styles on the test below. In commcare you can have more than one <menu> element for an ID to enable suite files to inject things into menus as extensions. In the event that two (or more) menus for an ID define different @style attributes for the same menu Id, it'll fall back to using neither.